src/components/SolvroLogo.tsx

```diff
@@ -0,0 +1,15 @@
+import Image from "next/image";
+
+export const SolvroLogo = () => {
+  return (
+    <a href="https://planer.solvro.pl/" className="inline-block">
```

Comment:
raczej powinniśmy dać tu po prostu, żeby też działalo lokalnie
```suggestion
    <Link href="/" className="inline-block">
```

Reply:
wiem, że ty tylko do pliku to wyrzuciłeś, ale jak możesz to popraw 😭